README.md

```diff
@@ -148,6 +191,14 @@ In the following table we present the ImageNet validation results for these mode
 | ResNet-18 [3]             | 69.86           | 69.63           |69.53|
 | SqueezeNet 1.1 [3]        | 58.128          | 57.678          ||
 
+### Keras
+MCT can quantize an existing 32-bit floating-point model to an 8-bit fixed-point (or less) model without compromising accuracy. 
+Below is a graph of [MobileNetV2](https://keras.io/api/applications/mobilenet/) accuracy on ImageNet vs average bit-width of weights (X-axis), using 
+single-precision quantization, mixed-precision quantization, and mixed-precision quantization with GPTQ. 
+
+<img src="https://github.com/sony/model_optimization/raw/main/docsrc/images/mbv2_accuracy_graph.png">
+
+For more results, please see [1]
```

Comment:
This is a very old not very relevant reference (for results purposes).
If we need more results maybe we can try to create a separate table and reference to it, but I'm not sure that the results in this paper are representing for the current performance of MCT